webapp/pages/post/_id/_slug/index.vue

```diff
@@ -170,6 +172,9 @@ export default {
     },
   },
   methods: {
+    reply(message) {
+      this.$refs.commentForm && this.$refs.commentForm.reply(message)
+    },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-22T15:19:58Z" title="Wednesday, January 22nd 2020, 4:19:58 pm +01:00">Jan 22, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -165,6 +172,9 @@ export default {
     },
   },
   methods: {
+    reply(message) {
+      this.$refs.commentForm && this.$refs.commentForm.reply(message)
+    },
```


Love this!